extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubePlaylistExtractor.java

```diff
@@ -74,7 +78,16 @@ public void onFetchPage(@Nonnull final Downloader downloader) throws IOException
                         .done())
                 .getBytes(StandardCharsets.UTF_8);
 
-        browseResponse = getJsonPostResponse("browse", body, localization);
+
+        final Map<String, List<String>> headers = new HashMap<>();
+        // Cookie is required due to consent
+        addYouTubeHeaders(headers);
+        headers.put("Content-Type", singletonList("application/json"));
+
+        final Response response = getDownloader().post(YOUTUBEI_V1_URL + "browse" + "?key="
+                + getKey() + DISABLE_PRETTY_PRINT_PARAMETER, headers, body, localization);
+
+        browseResponse = JsonUtils.toJsonObject(getValidJsonResponseBody(response));
```

Comment:
I think this part is not needed anymore since `addYouTubeHeaders` is now included in `getJsonPostResponse`:
<img width="479" alt="image" src="https://user-images.githubusercontent.com/36421898/210154129-5b5c8854-beb4-439e-bb14-984075082154.png">